posthog/settings/temporal.py

```diff
@@ -38,3 +38,4 @@
 CLICKHOUSE_MAX_BLOCK_SIZE_OVERRIDES: dict[int, int] = dict(
     [map(int, o.split(":")) for o in os.getenv("CLICKHOUSE_MAX_BLOCK_SIZE_OVERRIDES", "").split(",") if o]  # type: ignore
 )
+CLICKHOUSE_OFFLINE_5MIN_CLUSTER_HOST: str | None = os.getenv("CLICKHOUSE_OFFLINE_5MIN_CLUSTER_HOST", None)
```

Comment:
Maybe the `OFFLINE` naming is not needed, since in EU the host will be a different one, a coordinator in this case. Anyway, it's a silly naming thing, apply it if you think it makes sense!